### PR TITLE
Add a blacklist mechanism for statistics, fix #64

### LINKED ISSUE #64: Add a blacklist mechanism for statistics
As a maintainer, I want to restrict logged API calls introduced in https://github.com/etalab/udata/pull/61 to avoid getting spammed by our own polling.

### PR DESCRIPTION
**Pull request by davidbgk**

### CODE REVIEW

File: udata/api/__init__.py [hunk 1]
```diff
@@ -193,8 +193,11 @@ def extract_name_from_path(path):
 
 @apiv1.after_request
 def collect_stats(response):
-    action_name = extract_name_from_path(request.full_path)
-    if not current_app.config['TESTING']:
+    path = request.full_path
```
Comment (noirbizarre) [inline udata/api/__init__.py]: Do we blacklist URLs or endpoints with ``request.endpoint`` ?
```python
PIWIK_BLACKLIST = [
    '/api/1/me/notifications/
]
# vs
PIWIK_BLACKLIST = [
    'api.notifications'
]
```

File: udata/api/__init__.py [hunk 2]
```diff
@@ -193,8 +193,11 @@ def extract_name_from_path(path):
...
+    action_name = extract_name_from_path(path)
+    blacklist = current_app.config.get('PIWIK_BLACKLIST', [])
```
Comment (noirbizarre) [inline udata/api/__init__.py]: Maybe TRACKING_BLABKLIST will be better.

**Comment by davidbgk** [comment]
@noirbizarre updated

**Comment by noirbizarre** [comment]
r+wc

**Pull request merged**